src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -238,6 +243,11 @@ public enum PhaseShifterControlMode {
         INCREMENTAL
     }
 
+    public enum ReportedFeatures {
+        NEWTON_RAPHSON_LOAD_FLOW,
+        NEWTON_RAPHSON_SA,
```

Comment:
SA might be confusing here as traditionally it means "Security Analysis" but looking at your implementation it is used for both security analysis and sensitivity analysis (only in sensitivity analysis in fact before my push) . Is it on purpose?